### DISCUSSION TITLE
Subscriber e-mail not accepted?

### DISCUSSION THREAD

**Discussion by pharaofranz** [chime-frb-open-data/community#38] [Q&A] [answered]
Heya,

I recently signed up for the VOEvent e-mails, using as 'subscriber' e-mail the address of a Slack-channel. I know that e-mail address works as I can send messages to the channel from my regular e-mail. I noticed that CHIME detected FRB20180916B recently but we did not receive any message. Makes me wonder if I misunderstood something or if our request got rejected because of the cryptic Slack-generated e-mail address?

Thanks for the help!

Franz

**Comment by yniino** [comment]
Hi, 

I also sent a subscription request for both email and VOEvent via the web form on Oct. 21 (with an ordinary institute email address in my case). I haven't received any confirmation or any mail alerts since then (I have not set up comet yet). I understand you are receiving tons of subscription requests, but is there any way to confirm the status of my request? 

Thanks, 
Yuu

**Reply by andrewzwaniga** [reply]
Hi Yuu,

Sorry for the inconvenience, we were having a problem with our website. Your subscription was not registered in our database unfortunately. Please try filling the subscription webform again and it should work. Once you've submitted, let me know and I can confirm for you.

Best regards,
Andrew

**Reply by yniino** [reply]
Hi Andrew, 

Thank you for the message. I have just re-submitted the request (3rd trial). Please confirm the status. I also made my 2nd trial on Monday (around 9:00 UTC). Please discard one of them if the requests are duplicated. 

Thanks, 
Yuu

**Reply by SiqiLiu43** [reply]
Hi Yuu,

I am a colleague of @andrewzwaniga and I work closely with the CHIME/FRB VOEvents subscription database. I have checked for your request in our database, however I do not think your request was registered successfully. Would you mind re-opening our [webpage](https://www.chime-frb.ca/voevents) and re-submitting your information? We recently re-deployed our webpage and that should fix this problem. Please let me know once you have completed your request and I can confirm if it is saved in our database.

I sincerely apologize for the inconvenience this has caused, we greatly appreciate your interest in CHIME/FRB VOEvents and thank you for your understanding :)

Regards,
Siqi

**Reply by yniino** [reply]
Hi Siqi, 

Thank you for the suggestion. I have just cleared the cache of the web browser, reloaded the webform, and re-submitted the request. Please confirm if the request is received. 

Thanks, 
Yuu

**Reply by andrewzwaniga** [reply]
Hi Yuu,

Thanks for cooperating with my colleague @SiqiLiu43, and thanks, Siqi, for helping to solve this. I can confirm that your subscription is registered in our database and you are on the allow-list for XMLs and the mailing list for emails. 

Best regards,
Andrew

**Reply by yniino** [reply]
Hi Andrew and Siqi, 

Thank you for handling the problem. Now I can receive the alerts. 

Yuu

**Comment by andrewzwaniga** [comment]
Hello Franz and Yuu,

Thank you for reporting this incident. My apologies for the inconvenience. Could you both please try **submitting the webform again**? It seems there was a problem on your first attempt. This may have been caused by missing or corrupt information during the first attempt, though it is not entirely clear yet.

Please be sure that you fill all required fields and the `reCAPTCHA` at the bottom of the form. Lastly, a prompt should appear on the screen after clicking `Submit` telling you that the submission was successful.

Do not hesitate to ask more questions and report potential bugs here, including screenshots, if you have difficulties when filling the form.

Thanks for your understanding, and thanks for your interest in our VOEvents!

Best regards,
Andrew

**Reply by pharaofranz** [reply]
Hi Andrew,

Thanks for looking into this! I re-subscribed and got the friendly pop-up saying that the subscription was successful. Not sure if you have the means/time to check this or whether we should just wait for an event?

Cheers,
Franz

**Reply by andrewzwaniga** [reply]
Hi Franz, 

My apologies for the difficulty you are experiencing. I am still not seeing your information in our database. It seems to me like you are doing everything correctly. There may be an issue with how the backend system processes the data received from the webform. 

Could you please fill the webform once more? This time please take screenshots before submitting and post them here. This way I can investigate whether there is a corner case in the selections you have made. Lastly, it may help to know what browser you are using, in case the issue is browser specific.

Thank you,
Andrew

**Reply by andrewzwaniga** [reply]
You mentioned you are using a different kind of email address -- what are you trying to use? I didn't think it would be a problem but please write it here and we can use that for further investigation.

**Comment by marina-berezina** [comment]
Hi Andrew,

I have the same problem. I subscribed yesterday using both my email address and IP address but I got no messages from the system. Could you please check my registration as well?

Thanks!

Marina Berezina (MPIfR)

**Reply by andrewzwaniga** [reply]
Hi Marina,

Thanks for your message. My apologies that the service is not working for you.

In fact, the problem discussed in this forum so far appears to affect all users the same. We are actively debugging and hope to have a hot-fix for our subscription webform very soon, so please stay tuned.

The problem appears to be not related to the data that users are submitting, but rather how it is handled after submission. Despite the confirmation message that appears after clicking `Submit`, there is a problem registering that data. We are working to fix it right now.

Best regards,
Andrew

**Comment by andrewzwaniga** [comment, marked as answer]
We have fixed the problem that was preventing subscriptions from being registered. Thanks for your patience and cooperation.

Unfortunately, responses are **not cached** and we ask that you please re-submit your responses. On your end, nothing will appear to have changed; namely, you should see the same message upon submission that everything was successful. I will therefore post an update in this forum in a little bit once you all have had time to re-submit, in order to confirm that your subscription was registered.

Once again, my apologies for the delay in setting up your subscriptions. Thanks for your interest in CHIME/FRB VOEvents and we look forward to providing FRB triggers for your follow-up campaigns!

Best regards,
Andrew

**Comment by marina-berezina** [comment]
Thanks, Andrew!

I re-submitted my form. Could you please confirm it?

Marina

**Reply by SiqiLiu43** [reply]
Hi Marina, I am a colleague of @andrewzwaniga and I work closely with the CHIME/FRB VOEvents subscription database. I can confirm that your subscription request has been successfully registered. I apologize for the inconvenience this has caused; thank you for your interest in CHIME/FRB VOEvents :)

Cheers,
Siqi

**Reply by pharaofranz** [reply]
Hi Siqi, third time's a charm :-) -- can you check if my registration also went through this time? 

Thanks a lot for taking the time to fix things! Great work! @andrewzwaniga and @SiqiLiu43 

Franz

**Reply by marina-berezina** [reply]
Thanks a lot, Siqi!

**Reply by SiqiLiu43** [reply]
Hi Franz @pharaofranz , thank you for your patience :) I can confirm that your request has been registered successfully (under the email address of a slack channel).

Cheers,
Siqi

**Reply by pharaofranz** [reply]
Wicked! Keep 'em comin' them FRBs :-D 

Franz

**Comment by marina-berezina** [comment]
I am sorry! Should I already have received any alerts? I didn't have anything.

**Reply by andrewzwaniga** [reply]
Hi Marina, we have not published any alerts since your subscription was activated. Rest assured, you have not missed anything since then.

I added your email to our mailing list, and the IP address for your broker will be added to the allow-list shortly.

**Reply by marina-berezina** [reply]
Hi Andrew,

Thanks!
I just got a test event but it was not saved though I added flags: --save-event --save-event-directory=CHIME/

2021-11-10T15:27:42+0000 [INFO -] Subscribing to remote broker chimefrb.physics.mcgill.ca:8099
2021-11-10T16:23:52+0000 [INFO VOEventSubscriber,client] VOEvent ivo://azwaniga/test#TestEvent-2021-11-10T16:23:52Z received from IPv4Address(type='TCP', host='132.206.126.216', port=8099)
2021-11-10T16:28:52+0000 [INFO VOEventSubscriber] No iamalive received from IPv4Address(type='TCP', host='132.206.126.216', port=8099) for 120 seconds; disconecting
2021-11-10T16:28:52+0000 [INFO VOEventSubscriberFactory] Connection to IPv4Address(type='TCP', host='chimefrb.physics.mcgill.ca', port=8099) lost; will retry in 1 second

And is it correct when the service disconnects and then connects by itself?

**Reply by andrewzwaniga** [reply]
That is good news. It is a current curiosity as to why these disconnection-reconnection events happen, but they appear benign. Your broker should reconnect within a few seconds.

If you observe long-standing disconnections or rapidly fluctuating connection, please let us know. We are always working to improve the service, and robust connection is an area of concern.

**Reply by marina-berezina** [reply]
Now the disconnections are longer. Is it a problem from your side or from mine? Also are these test events empty?

2021-11-10T21:46:43+0000 [twisted.scripts._twistd_unix.UnixAppLogger#info] twistd 19.10.0 (/usr/bin/python 2.7.12) starting up.
2021-11-10T21:46:43+0000 [twisted.scripts._twistd_unix.UnixAppLogger#info] reactor class: twisted.internet.epollreactor.EPollReactor.
2021-11-10T21:46:43+0000 [INFO -] Subscribing to remote broker chimefrb.physics.mcgill.ca:8099
2021-11-10T21:47:13+0000 [INFO VOEventSubscriberFactory] Connection to IPv4Address(type='TCP', host='chimefrb.physics.mcgill.ca', port=8099) failed; will retry in 1 second
2021-11-10T21:47:46+0000 [INFO VOEventSubscriberFactory] Connection to IPv4Address(type='TCP', host='chimefrb.physics.mcgill.ca', port=8099) failed; will retry in 3 seconds
2021-11-10T21:48:26+0000 [INFO VOEventSubscriberFactory] Connection to IPv4Address(type='TCP', host='chimefrb.physics.mcgill.ca', port=8099) failed; will retry in 9 seconds
2021-11-10T21:49:20+0000 [INFO VOEventSubscriberFactory] Connection to IPv4Address(type='TCP', host='chimefrb.physics.mcgill.ca', port=8099) failed; will retry in 24 seconds

**Reply by andrewzwaniga** [reply]
Does your connection appear stable now? From my end I see your broker is connected.
